File: Lv1/Summer:Winter Coding(~2018) 소수만들기.py
from itertools import combinations

def solution(nums):
    temp = []
    answer = 0
    for i in combinations(nums, 3):
        for j in range(2,i[0]+i[1]+i[2]):
            if (i[0]+i[1]+i[2]) % j == 0:
                break
            if j == (i[0]+i[1]+i[2]) - 1:
                answer +=1

    return answer

File: lv1 Swift/Summer:Winter Coding(~2018) 소수 만들기.swift
import Foundation

func solution(_ nums:[Int]) -> Int {
    var answer = 0
    
    for i in 0..<nums.count {
        for j in i+1..<nums.count {
            for k in j+1..<nums.count {
                let num = nums[i]+nums[j]+nums[k]
                for n in 2..<num {
                    if num % n == 0{
                        break
                    }
                    if n == num - 1 {
                        answer += 1
                    }
                }
            }
        }
    }

    return answer
}
